vllm/model_executor/models/pixtral.py

```diff
@@ -115,8 +122,9 @@ def merge_multimodal_embeddings(input_ids: torch.Tensor,
     assert (D_txt == D_img), (f"Text features dim {D_txt} should be equal "
                               "to image features dim {D_img}")
     assert (seq_len == N_txt +
-            N_img), (f"seq_len {seq_len} should be equal to N_txt + N_img "
-                     f"{(N_txt, N_img, image_locations.sum().item())}")
+            N_img), (f"seq_len should be equal to N_txt + N_img, "
+                     f" but got {seq_len=} != {N_txt=} + {N_img=}, "
+                     f" for {image_locations.sum().item()} img locs")
```

Comment:
By the way, is it intended that you use a separate function than the built in one in `models/utils.py`?

Reply:
Nice catch - I think we can safely delete our custom merge function here